frontend/src/component/ui/Toggle.scss

```diff
@@ -12,7 +13,11 @@ $toggle-shrink-size: 4px;
   display: flex;
   justify-content: space-between;
   align-items: center;
-  width: 50%;
+  width: 60%;
```

Comment:
Just wondering. What is this width for?

Reply:
It's the size of the React component's top-level `div` element. (Which encompasses both the label and the button currently)

![image](https://user-images.githubusercontent.com/3276350/79054574-1911ab80-7c14-11ea-859a-af78ca13ec56.png)

Reply:
If you increase the width to say `80%`, the toggle will be further away from the label.

Reply:
With my latest commit, the `width` property is no longer a part of the `Toggle` component. This should hopefully allow the toggle to be easily added to more sections of the site.

Reply:
@Coteh Amazing! Thank you!!!

Reply:
@Coteh I like the way you think!